src/main/java/com/epam/spring/homework3/controller/ClientController.java

```diff
@@ -0,0 +1,54 @@
+package com.epam.spring.homework3.controller;
+
+import com.epam.spring.homework3.dto.ClientDTO;
+import com.epam.spring.homework3.service.ClientService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Slf4j
+@RestController
+@RequestMapping("/api")
+@RequiredArgsConstructor
+public class ClientController {
+    private final ClientService clientService;
+
+    @ResponseStatus(HttpStatus.OK)
+    @GetMapping(value = "/client/{id}")
+    public ClientDTO getClient(@PathVariable int id) {
+        log.info(this.getClass().getSimpleName() + ". Inside getClient method");
+        return clientService.getClient(id);
+    }
+
+    @ResponseStatus(HttpStatus.OK)
+    @GetMapping(value = "/client")
```

Comment:
client -> clients

Reply:
Same applies to other plural response entities (dishes, orders). That makes APIs closer to real-world and easier to understand )